libs/editor/WordPressEditor/src/main/res/layout/fragment_editor.xml

```diff
@@ -42,7 +42,7 @@
                 android:textColorHint="@color/sourceview_placeholder_text"
                 android:inputType="textCapSentences|textAutoCorrect"
                 android:imeOptions="flagNoExtractUi"
-                editor:fontFile="Merriweather-Bold.ttf"/>
```

Comment:
Since we dropped Merriweather Bold here, should we add `android:textStyle="bold"`?